radixdlt/src/main/java/com/radixdlt/environment/RemoteEventDispatcher.java

```diff
@@ -19,11 +19,19 @@
 
 import com.radixdlt.consensus.bft.BFTNode;
 
+import java.util.Set;
+
 /**
  * Attempts to dispatch an event to a remote node.
  *
  * @param <T> the event class
  */
 public interface RemoteEventDispatcher<T> {
 	void dispatch(BFTNode receiver, T t);
+
+	default void dispatch(Set<BFTNode> receivers, T t) {
```

Comment:
Perhaps it worth to use `Iterable` instead of `Set`. All we need from this container is the ability to iterate over elements contained inside. Other `Set` properties like uniqueness of elements are irrelevant here.

Reply:
yeah, definitely